kuma/users/models.py

```diff
@@ -195,6 +197,9 @@ def create_user_profile(sender, instance, created, **kwargs):
 
 @receiver(user_signed_up)
 def on_user_signed_up(sender, request, user, **kwargs):
+    context={'request':request}
+    msg='You have completed the first step of <a href="%s">getting started with MDN</a>' % wiki_url(context, 'MDN/Getting_started')
```

Comment:
Thanks! I have added a new commit. Looking for any options that allow squashing.